### ISSUE TITLE
Feature request/Discussion: have all the commands follow a format

### ISSUE THREAD

**Issue by vanessayuenn** [Rotonde/rotonde-client#59]
I propose the following format to standardize all currently available commands: 
```
/command param1 param2 ... paramN
```
So it will be less confusing for end users, and easier for future development. We can then also implement the autocomplete for commands like we have for @ mentions.

**Comment by aeonofdiscord** [comment]
Yeah, at the rate new commands are being added I think it makes sense to namespace them so people don't trip over them by accident.

**Comment by lsjroberts** [comment]
Would also mean you could hotkey it to `/`, so you can enter a command without having to click in the box.

**Comment by webdesserts** [comment]
I would like to see a consistent command pattern as well. Here is what I recommend based on the current set of commands:

## Commands

    /follow <$url>
    /unfollow <$url>
    /filter <$name>
    /filter clear
    /edit <$field> <$value>
    /edit <$entry> <$message>
    /delete <$field>
    /delete <$entry>
    /whisper <$user> <$message>
    /quote <$user> <$entry> [$message]

## Message Symbols
I think one thing that will help keep these commands simple (and possibly reduce the number of commands) is to make sure that pretty much anything that can be attached to a message has a corresponding symbol. This includes quotes, media, links, etc.. etc...

- **group** – `#rotonde`
- **mention** – `@neauoire`
- **quote** – `@neauoire:37`
- **media** – `file://corgi-butt.gif` where `corgi-butt.gif` is located under `/media`
- **link** – `dat://rotonde.hashbase.io` or `https://rotonde.hashbase.io`

----

Feedback is welcome.

**Comment by aeonofdiscord** [comment]
Not totally sure about using `file://` for media, since it looks like a valid URL but isn't one. The current setup uses `>>` (signifying "attachment", I guess) so maybe that or something close to it?

**Comment by eelfroth** [comment]
I happen to like the commands as they are. how about a configuration json that lets us define custom keybindings and command-aliases?

**Comment by webdesserts** [comment]
> Not totally sure about using file:// for media, since it looks like a valid URL but isn't one. The current setup uses >> (signifying "attachment", I guess) so maybe that or something close to it?

@aeonofdiscord Yeah, I see your point about the url issue. I have some more thoughts on all of this that I'm going to mull over. I think a lot of this depends on who we're catering to and what we want the final interface to look like.

**Comment by neauoire** [comment]
The idea here is that media would be wrapped in `{%path.jpg%}` to follow `{*bold*}` and `{_italic_}`.
